blackjack/blackjack_test.go

```diff
@@ -112,83 +136,84 @@ func TestPlayerBust(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	g.Player.Action = blackjack.ActionHit
+	players := []blackjack.Player{
+		{Name: "Planty"},
+		{Name: "Kevin"},
+		{Name: "Donald"},
+	}
+
+	g.Players = players
+
+	g.Players[0].Action = blackjack.ActionStand
+	g.Players[1].Action = blackjack.ActionStand
+	g.Players[2].Action = blackjack.ActionHit
+
 	g.Start()
 
-	got := blackjack.ReportMap[g.Player.HandOutcome]
+	output := &bytes.Buffer{}
+
+	g.Outcome(output)
 
-	want := "***** Bust!  Player loses *****"
+	want := "BlackjackWinBust"                                                                                       //"Planty: ***** Blackjack!  Player wins *****\nKevin: ***** Player wins! *****\nDonald: ***** Bust!  Player loses *****\n"
+	got := g.Players[0].HandOutcome.String() + g.Players[1].HandOutcome.String() + g.Players[2].HandOutcome.String() //output.String()
```

Comment:
We should be comparing named constants, not string-matching, shouldn't we? String matching creates very fragile tests which make it annoying to change a dot or a comma anywhere in our human-readable output.